README.md

```diff
@@ -178,6 +178,6 @@ latency: 99.9% < 10ms
 
 Kvrocks is under the Apache License Version 2.0. See the LICENSE file for details.
 
-## 微信公众号
```

Comment:
This information is mainly for Chinese users and thus I don't think it has to be changed.

Reply:
agree with @PragmaTwice